### PR TITLE
Support reading cluster configurations that are .yaml

### PR DESCRIPTION
**Pull request by treydock**

### CODE REVIEW

File: lib/ood_core/clusters.rb [hunk 1]
```diff
@@ -27,10 +27,10 @@ def load_file(path)
             end
           end
         elsif config.directory?
-          Pathname.glob(config.join("*.yml")).select(&:file?).select(&:readable?).each do |p|
+          Pathname.glob([config.join("*.yml"), config.join("*.yaml")]).select(&:exist?).select(&:readable?).each do |p|
```
Comment (ericfranz) [inline lib/ood_core/clusters.rb]: Why change `.select(&:file?)` to `.select(&:exist?)`?

Comment (treydock) [reply lib/ood_core/clusters.rb]: In the rare case that someone uses symlinks.

Comment (MorganRodgers) [reply lib/ood_core/clusters.rb]: `Pathname#file?` handles symlinks to files and directories transparently. This opens us up to the even less likely case that someone creates a directory named `something.yaml`.

Comment (treydock) [reply lib/ood_core/clusters.rb]: Ah in that case I reverted back to using `file?`.

**Pull request merged**